pkg/config/deployment.go

```diff
@@ -40,6 +40,8 @@ type GenericDeploymentSpec struct {
 	// The maximum length of time to execute deployment before giving up.
 	// Default is 6h.
 	Timeout Duration `json:"timeout,omitempty" default:"6h"`
+	// List of encrypted secrets and targets that should be decoded before using.
+	Encryption *SecretEncryption `json:"encryption"`
```

Comment:
Tell me if you have a better field name for this.

Reply:
I feel `encryption` is enough appropriate 👍